airbyte-webapp/src/setupTests.tsx

```diff
@@ -4,3 +4,13 @@
 // learn more: https://github.com/testing-library/jest-dom
 import "@testing-library/jest-dom/extend-expect";
 import "@testing-library/jest-dom";
+import React from "react";
```

Comment:
@timroes please note - I moved this fix to be global instead of ServiceForm specific